examples/meta/src/regression/least_angle_regression_observable.sg

```diff
@@ -0,0 +1,53 @@
+File f_feats_train = csv_file("../../data/regression_1d_linear_features_train.dat")
+File f_feats_test = csv_file("../../data/regression_1d_linear_features_test.dat")
+File f_labels_train = csv_file("../../data/regression_1d_linear_labels_train.dat")
+File f_labels_test = csv_file("../../data/regression_1d_linear_labels_test.dat")
+
+#![create_features]
+Features features_train = features(f_feats_train)
+Features features_test = features(f_feats_test)
+Labels labels_train = labels(f_labels_train)
+Labels labels_test = labels(f_labels_test)
+#![create_features]
+
+#![preprocess_features]
+Transformer SubMean = transformer("PruneVarSubMean")
```

Comment:
minor: pls don't mix CamelCase and camel_case variable names

Reply:
IMO we should have one or two examples that are specifically covering the observable framework. E.g. one for observing model parameters and one for observing special parameters (xvalidation). Show the API to retreive, list observable parameters, etc. Everything that is needed. So those examples would look quite different to the usual cookbook examples.
So it is fine to have this separate, but please put it under its own folder that is called something like "framework"

Reply:
Okay then, I'll try to move them to a separate section.